udsoncan/j2534.py

```diff
@@ -259,7 +259,7 @@ def PassThruGetLastError(self):
         pErrorDescription = (ctypes.c_char * 80)()
         result = dllPassThruGetLastError(pErrorDescription)
 
-        return Error_ID(hex(result)), str(pErrorDescription.value)
+        return Error_ID(hex(result)), pErrorDescription.value.decode()

```

Comment:
Previosly retures byte object like this **b'Device is not opened'**.
Now returns string **Device is not opened**.